tasks/security_agent.py

```diff
@@ -381,6 +381,7 @@ def build_functional_tests(
     env["CGO_ENABLED"] = "1"
 
     build_tags = build_tags.split(",")
+    build_tags.append("test")
```

Comment:
Probably stupid question but: do you still need this one or was it a test?

Reply:
It's actually needed for https://github.com/DataDog/datadog-agent/blob/b8ddf6c3b704dfca2ce0ecb381bb8509ab0826e9/comp/core/workloadmeta/impl/workloadmeta_mock.go#L6